dask/highlevelgraph.py

```diff
@@ -527,7 +536,10 @@ def layer_info_dict(self):
                 info[key] = html.escape(str(val))
         if self.collection_annotations is not None:
             for key, val in self.collection_annotations.items():
-                info[key] = html.escape(str(val))
+                if (
+                    key != "chunks"
+                ):  # displaying chunks can be a little overwhelming in the HTML, so we
```

Comment:
Unfinished sentence. Suggest rephrasing to a shorter comment. Maybe also placing the comment one line before the if statement begins (that would be two lines, rather than 3)

Reply:
Oh! fixed it. :heavy_check_mark: